e2e/tests/kube_client.go

```diff
@@ -0,0 +1,43 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"os/exec"
+)
+
+type CommandResult struct {
```

Comment:
Does this struct need to be exposed outside of this package?

1. We can potentially downscope until it's necessary? Ditto for the internal types.